README.md

```diff
@@ -1,92 +1,53 @@
-
-=> https://github.com/biohackrxiv/bhxiv-gen-pdf/blob/master/example/logic/paper.md
-
-and the raw text version
-
-=> https://raw.githubusercontent.com/biohackrxiv/bhxiv-gen-pdf/master/example/logic/paper.md
-
-It *can* embed LaTeX because pandoc generates LaTeX from markdown as an intermediate step towards generating the final PDF.
-
-The easiest start is to take the files in ./example/doc and modify them. Don't change the file names as they are used to generate the final paper. In fact, you can copy these files into your own git repo and continue from there.
-
-We have a web interface that allows you to generate the PDF at
-
-=> http://preview.biohackrxiv.org/
-
-Paste in the base URL for your repo and the tool will find the paper.md.
-
-Notes:
-
-1. Do not paste the path to the paper itself - only the base repo URL.
-2. One repo can not contain multiple paper.md's. It will pick the first one it finds.
-3. For biohackathons it pays to add a template repo people can template from, e.g
-
-=> https://github.com/biohackrxiv/publication-template
-
-# Introduction
+This CLI tool generates PDF files complying with the specifications for [biohackrxiv.org](https://biohackrxiv.org/).
+The PDF is generated using [pandoc](https://pandoc.org/) with LaTeX templates starting from a markdown (MD) file.
+The PDF can then be uploaded to [biohackrxiv.org](https://biohackrxiv.org/).
 
+**NOTE:** alternatively you could generate the PDF using the [web-based upload tool](http://preview.biohackrxiv.org/).
 
 # Quick start
 
-BioHackrXiv use [pandoc flavored markdown](https://pandoc.org/MANUAL.html#pandocs-markdown). It is very simple. See
-
-=> https://github.com/biohackrxiv/bhxiv-gen-pdf/blob/master/example/logic/paper.md
-
-and the raw text version
-
-=> https://raw.githubusercontent.com/biohackrxiv/bhxiv-gen-pdf/master/example/logic/paper.md
-
-It *can* embed LaTeX because pandoc generates LaTeX from markdown as an intermediate step towards generating the final PDF.
+BioHackrXiv supports files formatted in [pandoc flavored markdown](https://pandoc.org/MANUAL.html#pandocs-markdown) or text files.
+See the following example files:
+- [Pandoc flavored MD formatted example](https://github.com/biohackrxiv/bhxiv-gen-pdf/blob/master/example/logic/paper.md)
+- [Raw text example](https://raw.githubusercontent.com/biohackrxiv/bhxiv-gen-pdf/master/example/logic/paper.md)
 
-The easiest start is to take the files in ./example/doc and modify them. Don't change the file names as they are used to generate the final paper. In fact, you can copy these files into your own git repo and continue from there.
+LaTex can also be embedded into the Pandoc flavored MD files.
 
-We have a web interface that allows you to generate the PDF at
+To start, you could use the [template files in our repository](https://github.com/biohackrxiv/bhxiv-gen-pdf/blob/master/example/logic/)
```

Comment:
It's better to point to the template repository here. https://github.com/biohackrxiv/publication-template

Also, in the rendered page, it fails to add a new line here. Maybe add two spaces after the link?